docs/content/docs/dev/table/sqlClient.md

```diff
@@ -174,7 +174,7 @@ Mode "embedded" (default) submits Flink jobs from the local machine.
                                                --pyExecutable
                                                /usr/local/bin/python3). The python
                                                UDF worker depends on Python 3.6+,
-                                               Apache Beam (version == 2.27.0), Pip
+                                               Apache Beam (version == 2.38.0), Pip
```

Comment:
Does the version of pip need to be changed?

Reply:
Should I also add `pip>=20.3` into `dev-requirements.txt` ?